hack/env.sh

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/bash
 
 RABBIT_HOST=${RABBITMQ_SERVICE_SERVICE_HOST:-coreapi-broker}
-CUCOS_HOST=${BC_SERVER_SERVICE_HOST:-coreapi-server}
+F8A_HOST=${BC_SERVER_SERVICE_HOST:-coreapi-server}
 POSTGRES_HOST=${PGBOUNCER_SERVICE_HOST:-coreapi-pgbouncer}
 ANITYA_HOST=${ANITYA_SERVICE_HOST:-anitya-server}
 
 RABBIT_PORT=${RABBITMQ_SERVICE_SERVICE_PORT:-5672}
-CUCOS_PORT=${BC_SERVER_SERVICE_PORT:-5000}
+F8A_PORT=${BC_SERVER_SERVICE_PORT:-5000}
 POSTGRES_PORT=${PGBOUNCER_SERVICE_PORT:-5432}
 ANITYA_PORT=${ANITYA_SERVICE_PORT:-5000}
 
 export CCS_CELERY_BROKER=amqp://guest@${RABBIT_HOST}:${RABBIT_PORT}
-export CCS_CUCOS_SERVER=http://${CUCOS_HOST}:${CUCOS_PORT}
+export CCS_F8A_SERVER=http://${F8A_HOST}:${F8A_PORT}
```

Comment:
The `CCS` prefix means cucos, so I'd get rid of it too.

Anyway, `CCS_CUCOS_SERVER` doesn't seem to be used anywhere, so it can be safely removed, I think.